Packs/FeedAutofocus/Integrations/FeedAutofocus/FeedAutofocus.py

```diff
@@ -182,15 +367,22 @@ def build_iterator(self, limit=None, offset=None):
                     }
                 })
 
-        return parsed_indicators
+        if "Sample Feed" in self.indicator_feeds:
+            parsed_indicators.extend(self.sample_http_request())
+
+        if limit:
+            parsed_indicators = parsed_indicators[int(offset): int(offset) + int(limit)]
```

Comment:
why do we need this limit?

Reply:
its there for the get_indicators_command, not for the feed itself

Reply:
lets add a comment for this